File: CS/MvpvmNavigation/Interfaces.cs
using DevExpress.XtraBars.Ribbon;

namespace MvpvmNavigation {
    public interface IRibbonModule {
        RibbonControl Ribbon { get; }
    }
    public interface ISupportTransitions {
        void StartTransition(bool forward, object waitParameter);
        void EndTransition();
    }
    public interface ISupportViewModel {
        object ViewModel { get; }
    }
}

File: VB/MvpvmNavigation/Interfaces.vb
Imports DevExpress.XtraBars.Ribbon

Namespace MvpvmNavigation
    Public Interface IRibbonModule
        ReadOnly Property Ribbon() As RibbonControl
    End Interface
    Public Interface ISupportTransitions
        Sub StartTransition(ByVal forward As Boolean, ByVal waitParameter As Object)
        Sub EndTransition()
    End Interface
    Public Interface ISupportViewModel
        ReadOnly Property ViewModel() As Object
    End Interface
End Namespace